frontend/src/metabase/core/components/AccordionList/AccordionListCell.jsx

```diff
@@ -120,6 +120,7 @@ export const AccordionListCell = ({
     content = (
       <ListCellItem
         data-testid={itemTestId}
+        aria-label={name}
```

Comment:
Make this component more accessible and able to be found by `*ByRole("option", {name: "xxx"})`